cart/application/cartService.go

```diff
@@ -54,11 +54,12 @@ type (
 
 	// QtyAdjustmentResult restriction result enriched with the respective item
 	QtyAdjustmentResult struct {
-		OriginalItem      cartDomain.Item
-		DeliveryCode      string
-		WasDeleted        bool
-		RestrictionResult *validation.RestrictionResult
-		NewQty            int
+		OriginalItem          cartDomain.Item
+		DeliveryCode          string
+		WasDeleted            bool
+		RestrictionResult     *validation.RestrictionResult
+		NewQty                int
+		HasRemovedCouponCodes bool
```

Comment:
maybe other naming according to `WasDeleted`

`WereCouponCodesRemoved`
`CouponsCodesWereRemoved`
`RemovedCouponCodes`

Reply:
Personally i feel that the current naming is the closest compared to `WasDeleted.`
`RemovedCouponCodes` is the counterpart to `Deleted`, isnt it?